src/views/Workflow.vue

```diff
@@ -191,6 +191,13 @@ export default {
       this.$workflowService.unsubscribe(subscriptionId)
     })
   },
+  mounted () {
+    // Create a Tree View for the current workflow by default
+    const subscriptionId = this.subscribe('tree')
+    this.$nextTick(() => {
```

Comment:
If added to `created`, it would fail as the component was created, but nothing was added to the DOM, not even the Lumino panel holder.

On `mounted` we will have the component mounted on the DOM, but no guarantee that the Lumino objects are there too. So we wait one tick, then we can add to the widget :+1: